src/main/java/org/repodriller/scm/SubversionRepository.java

```
@@ -266,6 +266,12 @@ private String getDiffText(SVNRepository repository, SVNURL url, SVNLogEntryPath
 			return "";
 		}
 	}
+	
+	@Override
+	public List<Modification> getDiffBetweenCommits(String priorCommit, String laterCommit) {
+		// TODO Not yet implemented for SVN.
```

TBH I'd rather throw a RepoDrillerException. Fail hard.

---

Or return null. But don't lie.

---

Lol